docs/pages/dsHost/ds-host_L3_TestProcedure.md

```diff
@@ -0,0 +1,203 @@
+
+```
+
+#### Test Setup Configuration File
+
+Example Test Setup configuration File: `ut/host/tests/dsHost_L3_Tests/dsHost_L3_testSetup.yml`
+
+Update the artifact paths from which the binaries should be copied to the device.
+
+Set the execution paths and provide the stream paths for each test case.
+
+If a test case requires multiple streams or needs to be validated using several streams, ensure that all necessary streams are added sequentially for that specific test case.
+
+```yaml
+dsHost:
+  description: "dsHost Device Settings test setup"
+  assets:
+    device:
+      defaults: &defaults
+        artifacts:
+          - "<path>/bin/hal_test"
+          - "<path>/bin/libut_control.so"
+          - "<path>/bin/Sink_HostSettings.yaml"
+          - "<path>/bin/run.sh"
+        execute:
+          - "chmod +x /opt/HAL/dshost_L3/hal_test"
+          - "chmod +x /opt/HAL/dshost_L3/run.sh"
+          - cp -rf /usr/lib/libdshal.so /opt/HAL/dshost_L3/
+          - "ln -s /usr/lib/libds-hal.so /opt/HAL/dshost_L3/libdshal.so"
+        streams:
+        test1_VerifyConnect_Callback:
+          <<: *defaults
+          streams:
+```
+
+#### Test Suite Configuration
+
+Example Test Setup configuration File: `ut/host/tests/dsClasses/dshost_test_suite.yml`
+
+Update the execute command according to the device path where `HAL` binaries are copied.
+
+```yaml
+dsHost:
+  description: "dsHost Device Settings testing profile"
+  test:
+    execute: "/tmp/run.sh -p /tmp/Sink_HostSettings.yaml"
+    type: UT-C  # Cunit tests (UT-C)
+
+```
+
+## Run Test Cases
+
+Once the environment is set up, you can execute the test cases with the following command
+
+```bash
+python <TestCaseName.py> --config </PATH>/ut/host/tests/configs/example_rack_config.yml --deviceConfig </PATH>/ut/host/tests/configs/deviceConfig.yml
+```
+
+## Test Setup Connections
+
+To verify the verify the get temperature works as expected.
+For Example:
+
+- Get the temperature, and then place the device in a heat chamber, and verify the temperature increases.
+
+## Test Cases
+### dsHost_test1_VerifyTemperature.py
+
+#### Platform Support - test01
+
+- Sink/Source
+
+#### User Input Required - test01
+
+**Yes**: User interaction is necessary to connect/disconnect Composite Source device (This will be automated later).
+
+#### Acceptance Criteria - test01
+
+The temperature is noted to be increasing once placed in the heat chamber, or location to increase the temperature
+
+#### Expected Results - test01
+
+The temperature should increase
+
+#### Test Steps - test01
+
+- Initiate the Test:
+
+  - Select and execute the Python file: **`dsHost_test1_VerifyTemperature.py`**
+  - The test will automatically download all required artifacts, copying them to the designated target directory before commencing execution.
+
+- Increase Temperature prompt:
+
+    The test will Request User to place the device in a heat chamber, or do something to icnrease the temperature:
+
```

Comment:
Should press enter to consider the first reading and increase the temperature

Reply:
done